js/app/modules/card/title.js

```diff
@@ -41,6 +42,22 @@ const Title = new Module.Class({
             'Decorate on highlight', 'Whether to draw a custom decoration when the card is highlighted',
             GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT_ONLY,
             false),
+
+        /*
+         * Propety: lines
```

Comment:
Thanks, good to know. And my bad on the name change -- I originally just called it "lines" but figured it might make more sense to call it max-title-lines, and forgot to change it in the documentation. 

What do you think about that, actually? Maybe it should just be "lines" to be consistent with other places we use it?